src/fontra/client/lang/zh-CN.json

```diff
@@ -158,6 +158,11 @@
   "sidebar.selection-transformation.origin.top.center": "原点为上侧中心",
   "sidebar.selection-transformation.origin.top.left": "原点为左上角",
   "sidebar.selection-transformation.origin.top.right": "原点为右上角",
+  "sidebar.selection-transformation.pathOperations": "Path Operations",
```

Comment:
The style for keys is hyphenated, please use `sidebar.selection-transformation.path-operations`. Same below.

Reply:
(And in English as well of course)

Reply:
ok. done.